addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -50,7 +50,9 @@ export class KanbanRenderer extends Component {
 
     setup() {
         this.dialogClose = [];
+        /** @type {{ processing: string[], columnQuickCreateIsFolded: boolean }} */
         this.state = useState({
+            processing: [],
```

Comment:
not sure to understand why this is necessary. Note that I didn't read the whole diff of this commit